flang/test/Lower/Intrinsics/gepid.f90

```diff
@@ -0,0 +1,15 @@
+! RUN: bbc -emit-fir %s -o - | FileCheck %s
```

Comment:
Probably test the HLFIR flow. Use `-emit-hlfir`.

Reply:
done